platform-sdk/platform-apps/tests/PlatformTestingTool/src/main/java/com/swirlds/demo/virtualmerkle/map/account/AccountVirtualMapKey.java

```diff
@@ -74,6 +77,19 @@ public void serialize(final SerializableDataOutputStream out) throws IOException
         out.writeLong(accountID);
     }
 
+    void serialize(final WritableSequentialData out) {
+        out.writeLong(realmID);
+        out.writeLong(shardId);
+        out.writeLong(accountID);
+    }
+
+    @Deprecated
+    void serialize(final ByteBuffer buffer) {
+        buffer.putLong(realmID);
+        buffer.putLong(shardId);
```

Comment:
Not that I think it's extremely important, but you renamed this variable to `shardID` in the other class :)

Reply:
OK, let me rename it here, too